From bdd74ea512c5398d6561a95e725b55ba58059eb5 Mon Sep 17 00:00:00 2001
From: Allen Wild <allenwild93@gmail.com>
Date: Sat, 1 Feb 2020 03:38:21 -0500
Subject: [PATCH] Makefile: remove prepare3 from dtbo target

RPi commit a08645cf4481aef68c3d2e23ce43c53052aab972 added a %.dtbo rule,
but it depends on the prepare3 target which was removed in upstream
commit a5139fb368d2639afc6a3ea7e444f22a3e8ed3be, causing a "no rule to
make target overlays/disable-bt.dtbo" error.
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 9473dc537143..66764ad6afe4 100644
--- a/Makefile
+++ b/Makefile
@@ -1239,7 +1239,7 @@ ifneq ($(dtstree),)
 %.dtb: include/config/kernel.release scripts_dtc
 	$(Q)$(MAKE) $(build)=$(dtstree) $(dtstree)/$@
 
-%.dtbo: prepare3 scripts_dtc
+%.dtbo: scripts_dtc
 	$(Q)$(MAKE) $(build)=$(dtstree) $(dtstree)/$@
 
 PHONY += dtbs dtbs_install dt_binding_check
-- 
2.25.0

